### PR TITLE
Ab 73 authentication in express

### PR DESCRIPTION
**Pull request by michlask**

### CODE REVIEW

**Comment by michlask** [comment]
Please first review AB-18

File: plugin.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+          const payload = req.body
+          const ret = await controller[route.action]({ params, query, payload}, res)
+          res.send(ret)
+        } catch (e) {
+          console.log(e);
+        }
+      }
+
+      if (route.method === "GET") {
+        if (options.auth) {
```
Comment (wojtek-krysiak) [inline plugin.js]: when someone passes auth via configuration - you create 2 routes? - shouldn't have been if.... else statement?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+## Examples
+
+In examples folder we prepared one working example:
+* [simplest integration with mongodb database](examples/simple.js)
+
+You can run one of them by typing:
+
+```bash
+PORT=3000 MONGO_URL=mongodb://localhost:27017/yourserver node examples/simple.js
+```
```
Comment (wojtek-krysiak) [inline README.md]: there is no mention about authentication?

File: examples/auth.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    databases: [connection],
+    branding: {
+      companyName: 'Amazing c.o.',
+    },
+    rootPath: adminRootPath,
+    auth: checkSignIn
+  }
+  const adminRouter = await AdminBroExpress.register(app, adminBroOptions)
+
+  app.get(adminLoginPath, async (req, res) => {
```
Comment (wojtek-krysiak) [inline examples/auth.js]: this should go to a plugin in a similar fashion as in hapijs

**Pull request merged**